website_and_docs/content/documentation/webdriver/bidi_apis.en.md

```diff
@@ -5,19 +5,56 @@ weight: 11
 aliases: ["/documentation/en/webdriver/bidi_apis/"]
 ---
 
-In Selenium 4, new Evented APIs were introduced that
-allow users to be able to capture events from the
-browser as they happen rather than using the
-traditional approach of Request/Response that
-WebDriver has used for other APIs.
+Selenium is working with Browser Vendors to create the 
+[BiDirectional WebDriver Protocol](https://w3c.github.io/webdriver-bidi/) 
+as a means to provide a stable, cross-browser implementation of a subset of the functionality found
+[Chrome DevTools](../../support_packages/chrome_devtools).
+The traditional webdriver model of strict request/response commands will be supplemented with the ability to
+stream events from the user agent to the controlling software via WebSockets, better matching the evented nature of the browser DOM.
 
-Internally WebDriver will create a WebSocket connection
-to the browser for events and commands to be transmitted.
+While this initiative is being developed, Selenium has implemented this API which is currently supported by
+Chrome, Edge and Firefox using the Chrome DevTools Protocol. 
+While the underlying implementation may change, the goal is for this API to remain constant.
 
 The following list of APIs will be growing as the Selenium
 project works through supporting real world use cases. If there
 is a missing API, please raise a [feature request](https://github.com/SeleniumHQ/selenium/issues/new?assignees=&labels=&template=feature.md).
 
+## Register Basic Auth:
+
+Some applications require to keep some pages behind an auth and most of the time
```

Comment:
"Some applications make use of browser authentication to secure pages. With Selenium, you can automate...."